index.html

```diff
@@ -22,7 +22,7 @@
                         <ul>
                             <li><a href="#welcome-section">About</a></li>
                             <li><a href="#projects">Work</a></li>
-                            <li><a href="#contact">Contact</a></li>
+                            <li><a href="#contact">Contact me</a></li>
                         </ul>
```

Comment:
Could you remove double links with the same name and link? This will make the code better and more user-friendly as a user does not expect repeated items.